tigerbeetle-hs.cabal

```diff
@@ -20,102 +21,96 @@ name:               tigerbeetle-hs
 -- PVP summary:     +-+------- breaking API changes
 --                  | | +----- non-breaking API additions
 --                  | | | +--- code changes with no API change
-version:            0.1.0.0
+version:         0.1.0.0
 
 -- A short (one-line) description of the package.
-synopsis:           A Haskell client library for Tigerbeetle database
+synopsis:        A Haskell client library for Tigerbeetle database
 
 -- A longer description of the package.
 -- description:
 
 -- URL for the project homepage or repository.
-homepage:           https://github.com/agentultra/tigerbeetle-hs
+homepage:        https://github.com/agentultra/tigerbeetle-hs
 
 -- The license under which the package is released.
-license:            BSD-3-Clause
+license:         BSD-3-Clause
 
 -- The file containing the license text.
-license-file:       LICENSE
+license-file:    LICENSE
 
 -- The package author(s).
-author:             James King
+author:          James King
 
 -- An email address to which users can send suggestions, bug reports, and patches.
-maintainer:         james@agentultra.com
+maintainer:      james@agentultra.com
 
 -- A copyright notice.
 -- copyright:
-category:           Database
-build-type:         Simple
+category:        Database
+build-type:      Simple
 
 -- Extra doc files to be distributed with the package, such as a CHANGELOG or a README.
-extra-doc-files:    CHANGELOG.md
-
--- Extra source files to be distributed with the package, such as examples, or a tutorial module.
-extra-source-files: include/tigerbeetle/src/clients/c/tb_client.h
-                  , include/tb_client_shim.h
-                  , include/tb_client_shim.c
```

Comment:
This is a big change, I don't think we need the shims anymore.

Reply:
Well, we might in the future if we need to do anything in C land, but for now it seems like we can get away without it.